runtime-sdk/src/modules/rofl/app/client.rs

```diff
@@ -185,9 +186,21 @@ where
 {
     fn new(state: Arc<processor::State<A>>, cmdq: mpsc::WeakSender<processor::Command>) -> Self {
         Self {
-            state,
             cmdq,
             latest_round: Arc::new(AtomicU64::new(0)),
+            rpc: Arc::new(RpcClient::new_runtime(
+                state.host.clone(),
+                ENCLAVE_RPC_ENDPOINT_RONL,
+                session::Builder::default()
+                    .use_endorsement(true)
+                    .quote_policy(None)
+                    .local_identity(state.identity.clone())
+                    .remote_enclaves(None),
+                8,
```

Comment:
A comment explaining the chosen session parameters would be nice.

Reply:
Adjusted and added a comment.